instrumentation/otel-webserver-module/src/core/api/RequestProcessingEngine.cpp

```diff
@@ -19,17 +19,23 @@
 #include "api/ApiUtils.h"
 #include "api/Payload.h"
 #include "sdkwrapper/SdkWrapper.h"
+#include "sdkwrapper/IScopedSpan.h"
 #include <sys/types.h>
 #include <unistd.h>
 #include <boost/filesystem.hpp>
 #include <boost/thread/locks.hpp>
 #include <log4cxx/logger.h>
 #include <unordered_map>
+#include <sstream>
 
 
 namespace appd {
 namespace core {
 
+#define HTTP_ERROR_1XX 100
+#define HTTP_ERROR_4XX 400
+#define HTTP_ERROR_5XX 500
```

Comment:
Just a discussion, Is it not better to use constexpr instead ?

Reply:
or we can use constexpr https://en.cppreference.com/w/cpp/language/constexpr as well.

Reply:
3 defines look a little clumsy. Should we use a different data structure(like a struct)?

Reply:
Addressed this by using constexpr in SdkConstants.h